fdk/event_handler.py

```diff
@@ -50,3 +54,16 @@ def event_handle(handle_code):
         )
 
     return pure_handler
+
+
+def log_frame_header(headers):
+    framer = os.environ.get(constants.FN_LOGFRAME_NAME)
```

Comment:
shouldn't this be at the module level?
both `framer` and `value_src` will have constant value through the lifetime of th container, meaning, we get retrieve them only once, there's no need to do this for each request.

Reply:
Good point. I've refactored to get the env values once at module level